### ISSUE TITLE
Installer showing false-success if any of game folder is not a folder

### ISSUE THREAD

**Issue by hibiyasleep** [marzent/XIV-on-Mac#90]
## What Happened
* I've symlinked `game/movie` to somewhere outer in my desktop installation (for some reason)
* I copied it from desktop over SMB – so link broke, `FFXIVApp.installed` = false
* there was **broken link `game/movie`** that blocks `mkdir` or any subdirectory access
* installer shows 'Complete', however logging in opens installer again

## Reproduce
```sh
#  any installation destination here
cd ~/Library/Application Support/XIV on Mac/ffxiv/game
mv movie movie_
touch movie
```
then log in, 'no installation found', choose installed folder, 'Complete', log in, …

---

I know this is damn rare and almost-never-gonna-be-happen, but imo, in this case Installer should fail and show correct I/O error message.

**Comment by marzent** [comment]
Hm indeed there were also failures due to broken movie symlinks when copying from an existing official Mac version of the game…

I fixed them (by just deleting broken symlinks) here https://github.com/marzent/XIV-on-Mac/blob/c33c5f35cf5ae52ca7515d9fef62f0707048d769/XIV%20on%20Mac/InstallerController.swift#L220 back then…

Actually I probably should extend that treatment to all to be extracted files, log the error, and reflect it in the installer end screen as well I think.

**Comment by marzent** [comment]
Should theoretically be fixed with 50ca0788f1ebe8760dde86eda6f210ea71023d1a, or at least log an error now. 

Will be changing the gui later so still leaving this open, could you see if it works for your use case as of 4.4?

**Comment by hibiyasleep** [comment]
yeah tested on 4.4.1, it just repairs broken `ffxiv/game/movie` symlink well. thank you!
